src/Coin.cpp

```diff
@@ -41,7 +41,7 @@
 #include "Oasis/Entry.h"
 #include "Ontology/Entry.h"
 #include "Polkadot/Entry.h"
-#include "Ripple/Entry.h"
+#include "XRP/Entry.h"
```

Comment:
Only Ripple/XRP is moved, why? Can it not be handled like the rest?

Reply:
Most coins are using "name" (or something similar) for the test directory name. Ripple's name is "XRP", so I renamed it as it's a simple change